src/vs/editor/contrib/suggest/browser/suggestController.ts

```diff
@@ -257,6 +259,9 @@ export class SuggestController implements IEditorContribution {
 			if (index === -1) {
 				index = 0;
 			}
+
+			this.completionItems = e.completionModel.items;
```

Comment:
We should not store `completionItems` in the controller. This is data of a particular invocation and not state. Also, the `_insertSuggestion` has already access to the model and all items